<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-QN1NF34" ignored="0" name="Dependency Test" tests="4" failures="1" timestamp="2020-06-05T10:26:44 CEST" time="0.044" errors="0">
  <testcase name="doUserreg" time="0.004" classname="TestCases.Testcsae3">
    <failure type="java.lang.AssertionError" message="User not registered successfully">
      <![CDATA[java.lang.AssertionError: User not registered successfully
at TestCases.Testcsae3.doUserreg(Testcsae3.java:12)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- doUserreg -->
  <testcase name="doLogin" time="0.0" classname="TestCases.Testcsae3">
    <skipped/>
  </testcase> <!-- doLogin -->
  <testcase name="thirdTest" time="0.003" classname="TestCases.Testcsae3"/>
  <testcase name="fourthTest" time="0.002" classname="TestCases.Testcsae3"/>
</testsuite> <!-- Dependency Test -->
